client/api/python/test/unit/test_client.py

```diff
@@ -42,6 +42,19 @@ def test_cluster(self):
         info = c.cluster_info(cluster['id'])
         self.assertEqual(True, info == cluster)
 
+        # change cluster flags
+        cluster_setflags_req = {}
+        cluster_setflags_req['block'] = False
+        cluster_setflags_req['file'] = True
+        ok = c.cluster_setflags(cluster['id'], cluster_setflags_req)
+        self.assertEqual(True, ok)
+
+        # verify the cluster flags have changed
+        info = c.cluster_info(cluster['id'])
+        self.assertEqual(info['id'], cluster['id'])
+        self.assertEqual(False, info['block'])
+        self.assertEqual(True, info['file'])
```

Comment:
Small nit. Python unittest has assertTrue and assertFalse methods that are more appropriate for testing booleans. However, since the existing test code is already doing this all over the place (and some even more uglier stuff) I don't think its worth complaining too much about.

Reply:
fixed, thanks